### ISSUE TITLE
How about cleargrad flag in loss.backward?

### ISSUE THREAD

**Issue by arisliang** [chainer/chainer#4695]
Why always accumulate the grad? What if adding a flag in backward to indicate whether to overwrite the previous grad, or accumulate?

**Comment by arisliang** [comment]
Don't see zerograds is much used anyway. So maybe it's not that relevant. Just thought might be an idea to have a overwrite / accumulate grad mode.
